cmd/flags.go

```diff
@@ -56,6 +57,13 @@ var algos = map[tpm2.Algorithm]string{
 	tpm2.AlgSHA512: "sha512",
 }
 
+var hierarchyNames = map[string]tpmutil.Handle{
+	"endorsement": tpm2.HandleEndorsement,
+	"owner":       tpm2.HandleOwner,
+	"platform":    tpm2.HandlePlatform,
+	"null":        tpm2.HandleNull,
+}
+
```

Comment:
Move this back to where it was (now that we don't need it here).

Reply:
done